transformers/component_tests/cat/configured_transformer_test.go

```diff
@@ -136,22 +145,26 @@ var _ = Describe("Executing the transformer", func() {
 			var wardsResult test_helpers.MappingResWithAddress
 			err := db.Get(&wardsResult, `SELECT diff_id, header_id, address_id, usr AS key, wards.wards AS value FROM maker.wards`)
 			Expect(err).NotTo(HaveOccurred())
-			Expect(wardsResult.AddressID).To(Equal(strconv.FormatInt(catAddressID, 10)))
+			Expect(wardsResult.AddressID).To(Equal(catAddressID))
 			test_helpers.AssertMapping(wardsResult.MappingRes, wardsDiff.ID, header.Id, strconv.FormatInt(userAddressID, 10), "1")
```

Comment:
Definitely not a blocker but could be cool to introduce a `AssertMappingWithAddress` helper so that we don't need to remember to always include a separate expectation for the `AddressID` if we're doing this more often

Reply:
Heh - I flip/flopped on that. I can't remember why I stayed with this version.